indico/modules/events/editing/operations.py

```diff
@@ -264,3 +265,21 @@ def update_file_type(file_type, **data):
 def delete_file_type(file_type):
     logger.info('File type %r deleted by %r', file_type, session.user)
     db.session.delete(file_type)
+
+
+def create_new_review_condition(event, editable_type, file_types):
+    review_condition = EditingReviewCondition(event=event, type=editable_type, file_types=file_types)
+    db.session.flush()
+    logger.info('Review condition %r created by %r', repr(review_condition), session.user)
```

Comment:
`%r` already uses the repr, so if you call `repr()` again it will look *quite* ugly